migrate/v0_14/migrate_test.go

```diff
@@ -111,15 +111,15 @@ func getAccount(accounts authexported.GenesisAccounts, address sdk.AccAddress) a
 }
 
 func TestIncentive(t *testing.T) {
-	bz, err := ioutil.ReadFile(filepath.Join("testdata", "kava-4-incentive-state.json"))
+	bz, err := ioutil.ReadFile(filepath.Join("testdata", "kava-6-incentive-state.json"))
 	require.NoError(t, err)
 	var oldIncentiveGenState v0_11incentive.GenesisState
 	cdc := app.MakeCodec()
 	require.NotPanics(t, func() {
 		cdc.MustUnmarshalJSON(bz, &oldIncentiveGenState)
 	})
 
-	bz, err = ioutil.ReadFile(filepath.Join("testdata", "kava-4-harvest-state.json"))
+	bz, err = ioutil.ReadFile(filepath.Join("testdata", "kava-6-harvest-state.json"))
```

Comment:
```suggestion
	bz, err = ioutil.ReadFile(filepath.Join("testdata", "kava-6-hard-state.json"))
```
Module names should line up even in tests

Reply:
Technically the module name is `harvest` in the kava-6 exported state, it's only `hard` after migration